Modules/Loadable/Sequences/Widgets/qMRMLSequenceBrowserPlayWidget.cxx

```diff
@@ -35,6 +36,7 @@ class qMRMLSequenceBrowserPlayWidgetPrivate
   Q_DECLARE_PUBLIC(qMRMLSequenceBrowserPlayWidget);
 protected:
   qMRMLSequenceBrowserPlayWidget* const q_ptr;
+  bool recordingControlsVisible;
```

Comment:
isRecordingControlsVisible ?

Reply:
Member variables in private implementation classes are all public and start with capital letter.